apps/condo/domains/notification/adapters/firebaseAdapter.js

```diff
@@ -0,0 +1,200 @@
+const admin = require('firebase-admin')
+const isEmpty = require('lodash/isEmpty')
+const isNull = require('lodash/isNull')
+const get = require('lodash/get')
+const faker = require('faker')
+
+const {
+    PUSH_FAKE_TOKEN_SUCCESS,
+    PUSH_FAKE_TOKEN_FAIL,
+    FIREBASE_CONFIG_ENV,
+} = require('../constants/constants')
+
+const EMPTY_CONFIG_ERROR = `Valid ${FIREBASE_CONFIG_ENV} should be provided within .helm (.env), and can be retrieved from https://console.firebase.google.com/project/__PROJECT_ID__/settings/serviceaccounts/adminsdk`
```

Comment:
Could you please extract errors into `constant/errors`

Reply:
done